ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/process/search_tree/Leaf.java

```diff
@@ -133,6 +137,12 @@ void evaluate(Network network, Crac crac, String referenceNetworkVariant, RaoPar
         this.status = Status.EVALUATION_RUNNING;
         String leafNetworkVariant;
 
+        if (isRoot()) {
+            LOGGER.info("SearchTreeRao: evaluate root leaf");
+        } else {
+            LOGGER.info("SearchTreeRao: evaluate network action(s) " + networkActions.stream().map(NetworkAction::getName).collect(Collectors.joining(", ")));
```

Comment:
For some reason, i my editor, SonarLint prefers when I create a variable containing the content of this info, and put directly this variable in LOGGER...

Reply:
done